### DISCUSSION TITLE
Source-build triage / planning

### DISCUSSION THREAD

**Discussion by MichaelSimons** [dotnet/source-build#3274] [Announcements]
The .NET source-build team along with some distro maintainers meet weekly to triage issues, plan work and discuss various topics.  Folks consuming or contributing to the project are welcome to join.  Feel free to bring any topics you have to discuss.

Meeting (30 mins):
* Thursdays - 9:00 am Pacific / 5:00 pm UTC
* [Teams link](https://teams.microsoft.com/l/meetup-join/19%3ameeting_MGY3ZWRiNTUtNjQzNi00OTczLWJiOGUtOWE1YTVlNWRiYTEx%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%228dae9f0e-155f-49b6-a2f2-3bd02ca469a5%22%7d) (you can use the browser)

**Comment by MichaelSimons** [comment]
cc @dotnet/distro-maintainers

**Comment by MichaelSimons** [comment]
This week's meeting on March 23rd is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on April 27th is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on May 11th is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on June 15th is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on July 15th is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on August 10th is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on August 31st is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on Sept 14th is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on Sept 21st is cancelled.

**Comment by MichaelSimons** [comment]
This meeting has been pushed back 30 minutes and shortened to 30 minutes.

**Comment by MichaelSimons** [comment]
This week's meeting on Nov 9th is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on Nov 23rd is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on Nov 29th is cancelled.

**Comment by MichaelSimons** [comment]
The Dec 21st and 28th meetings will be cancelled for the holiday break.

**Comment by MichaelSimons** [comment]
This week's meeting on Jan 25th is cancelled.

**Comment by MichaelSimons** [comment]
Today's meeting (on Feb 22th) will be moved up 1/2 hr to 9:00 am Pacific / 5:00 pm UTC.

**Comment by MichaelSimons** [comment]
This week's meeting on March 7th is cancelled.

**Comment by MichaelSimons** [comment]
Today's meeting (April 11th) is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on April 24th is cancelled.

**Comment by MichaelSimons** [comment]
Next week's meeting on June 13th is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on July 4th is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on July 18th is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting on Aug 15th is cancelled.

**Comment by mthalman** [comment]
This week's meeting on Aug 29 is cancelled.

**Comment by MichaelSimons** [comment]
Today's meeting (Oct 3) is cancelled.

**Comment by MichaelSimons** [comment]
Next week's meeting (Nov 28) is cancelled.

**Comment by MichaelSimons** [comment]
The meetings over the holiday season are cancelled - Dec 26 and Jan 2.

**Reply by MichaelSimons** [reply]
Cancelling today's meeting (Dec 19th) as there is only one non-urgent issue to triage.

**Comment by MichaelSimons** [comment]
Today's meeting (Feb 13) is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting (March 13) is cancelled.

**Comment by MichaelSimons** [comment]
This week's meeting (April 10) is cancelled.

**Comment by MichaelSimons** [comment]
Next week's meeting (June 26) is cancelled.

**Comment by MichaelSimons** [comment]
Today's meeting (Oct 23) is cancelled.

**Comment by MichaelSimons** [comment]
Next week's meeting (Nov 27) is cancelled.

**Comment by MichaelSimons** [comment]
The Dec 18th and Jan 1st meetings will be cancelled for the holiday break.

**Comment by MichaelSimons** [comment]
Today's meeting (Jan 15) is cancelled.
